pybutton/response.py

```diff
@@ -47,20 +45,27 @@ def prev(self):
     def __repr__(self):
         values = []
 
+        classPrefix = 'class pybutton.Response'
+
         if isinstance(self.response_data, dict):
             for k, v in self.response_data.items():
                 values = values + ['{0}: {1}'.format(k, v)]
-            return '<class pybutton.Response {0}>'.format(', '.join(values))
+            return '<{0} {1}>'.format(
```

Comment:
nit: because `values` is now only relevant to this if-statement, we can move the allocation inside.